doc/source/amazon/step-zero-aws.rst

```diff
@@ -3,183 +3,171 @@
+
+       kops create cluster $NAME \
+         --zones $ZONES \
+         --authorization RBAC \
+         --master-size t2.micro \
+         --master-volume-size 10 \
+         --node-size t2.medium \
+         --node-volume-size 10 \
+         --yes
+
+   For a more secure setup add the following params to the kops command::
+   
+         --topology private \
+         --networking weave \
+
+   This creates a cluster where all of the masters and nodes are in private subnets and don't have external IP addresses.  A mis-configured security group or insecure ssh configuration is less likely to compromise the cluster.
+   In order to SSH into your cluster you will need to set up a bastion node.  Make sure you do that step below.
+   If you have the default number of elastic IPs (10) you may need to put in a request to AWS support to bump up that limit.  The alternative is reducing the number of zones specified.
+   
+   More reading on this subject:
+   https://github.com/kubernetes/kops/blob/master/docs/networking.md
+
+   Settings to consider (not covered in this guide)::
+   
+       --vpc
+         Allows you to use a custom VPC or share a VPC
+         https://github.com/kubernetes/kops/blob/master/docs/run_in_existing_vpc.md
+       --master-count
+         Spawns more masters in one or more VPCs
+         This improves redudancy and reduces downtime during cluster upgrades
+       --master-zones
+         specify zones to run the master in
+       --node-count
+         Increases the total nodes created (default 2)
+       --master/node-security-groups
+         Allows you to specify additional security groups to put the masters and nodes in by default
+       --ssh-access
+         By default SSH access is open to the world (0.0.0.0).
+         If you are using a private topology, this is not a problem.
+         If you are using a public topology make sure your ssh keys are strong and you keep sshd up to date on your cluster's nodes.
+
+10. Wait for the cluster to start-up
+
+    Running the 'kops validate cluster' command will tell us what the current state of setup is.
+    If you see "can not get nodes" initially, just be patient as the cluster can't report until a
+    few basic services are up and running.
+   
+    Keep running 'kops validate cluster' until you see "Your cluster $NAME is ready" at the end of the output.
+   
+    ``time until kops validate cluster; do sleep 15 ; done`` can be used to automate the waiting process.
+    
+    If at any point you wish to destroy your cluster after this step, run ``kops delete cluster $NAME --yes``
+    
+
+11. Confirm that ``kubectl`` is connected to your Kubernetes cluster.
+
+    Run::
+
+       kubectl get nodes
+
+    You should see a list of two nodes, each beginning with ``ip``.
+    
+    If you want to run kubectl from a box not on AWS, you can use run the following on AWS: ``kubectl export kubecfg``
+       
```

Comment:
I think you want kops export kubecfg not 
kubectl.

Reply:
Thanks for catching that :+1: